src/components/routes/Account/Account.tsx

```diff
@@ -15,6 +14,7 @@ import AudioLoading from 'shared/loading/AudioLoading';
 import Divider from 'shared/Divider';
 import GeneralWrapper from 'shared/GeneralWrapper';
 import NotFound from 'routes/NotFound';
+import ReusableNavBar from 'shared/ReusableNavBar';
```

Comment:
maybe just `TabNavBar`? a component by definition should be `reusable`

Reply:
renamed